wolfcrypt/src/random.c

```diff
@@ -1910,15 +1910,25 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         {
             return RAN_BLOCK_E;
         }
+        (void)os;
     }
 
 
 #elif defined(FREESCALE_RNGA)
 
     int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
     {
-        RNGA_DRV_GetRandomData(RNGA_INSTANCE, output, sz);
-        return 0;
+        status_t status;
+        status = RNGA_GetRandomData(RNG, output, sz);
+        if (status == kStatus_Success)
+        {
+            return(0);
+        }
+        else
+        {
+            return RAN_BLOCK_E;
+        }
+        (void)os;
```

Comment:
Some compilers may not like the logic after return. Suggest the code looks like:

```
    status_t status = RNGA_GetRandomData(RNG, output, sz);
    if (status == kStatus_Success) {
        return 0;
    }
    (void)os;
    return RAN_BLOCK_E;
```